Utils/Dataflow/README

```diff
@@ -133,8 +133,34 @@ following requirements:
 
 [3] EOP, End-of-Process marker. Default value: '\0' (NULL)
 
+==================
+* 3. Stage types *
+==================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.).
+
+2. Transform stage - these stages perform a set of operations for different
+cases on metadata and form output messages according to a particular pattern
+(e.g. select relevant metadata from a raw dataset, append metadata
+to a message, prepare messages for upload).
```

Comment:
Maybe "... prepare a message for upload"? To follow the style of two previous examples.

Reply:
Maybe "preparation for upload" no to be mentioned here at all? I mean, we have "2.1. Pre-sink stage" section below -- it looks like a rightful place to talk about this for me...